config/config.go

```diff
@@ -65,6 +65,11 @@ type (
 			Cache    string        `envconfig:"DRONE_GC_CACHE" default:"10gb"`
 		}
 
+		Reaper struct {
+			Enabled  bool          `envconfig:"DRONE_ENABLE_REAPER" default:"false"`
```

Comment:
Enabled isn't used yet, right? It's currently based directly on the env variable without using this config.

Reply:
You're right, I've removed it for now